tests/unit/core/test_installAddon.js

```diff
@@ -200,6 +200,27 @@ describe(__filename, () => {
     sinon.assert.calledWith(_addonManager.getAddon, '@new-guid');
   });
 
+  it('sets status when add-on is loaded on update', () => {
+    const Component = componentWithInstallHelpers();
+    const _addonManager = getFakeAddonManagerWrapper({
+      getAddon: Promise.resolve({
```

Comment:
It's not necessary for this patch but at some point this should really resolve the final object using a helper. I see that sometimes [`type` is passed](https://github.com/mozilla/addons-frontend/blob/6a6a293a48620b477376e407e72f264562ad557c/tests/unit/core/test_installAddon.js#L163) so a helper would be better to manager the return value.